### DISCUSSION TITLE
在 antd 中运行 yarn start 去跑起来的时候，发现会报  These dependencies were not found:

### DISCUSSION THREAD

**Discussion by cl1069573062** [alibaba/formily#2405] [Q&A 2.x]
在 antd 中运行 yarn start 去跑起来的时候，发现会报 
These dependencies were not found:

* @formily/antd in ./src/.umi/dumi/demos/index.ts
* @formily/core in ./src/.umi/dumi/demos/index.ts
* @formily/react in ./src/.umi/dumi/demos/index.ts
* @formily/reactive in ./src/.umi/dumi/demos/index.ts
实际上在lerna的根项目中node_modules已经安装的有了， 但是antd目录下node_modules里面没有就会找不到，一定要在antd node_modules中有才可以， 这个是为啥呢

**Comment by coolbob1998** [comment]
yarn build 一下就行了
